avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2017 OVO Energy
```

Comment:
2017-2018 maybe?

Reply:
yup

Reply:
the sbt-header plugin does not support it.